lib/groups.sql

```diff
@@ -6,6 +6,19 @@ FROM
 WHERE
     gc.assessment_id = $assessment_id AND gc.deleted_at IS NULL;
 
+-- BLOCK get_creator_role
```

Comment:
To check my understanding: this gets a role that should be assigned to the creator, not the role or whoever created the group? I know that's a subtle distinction, but the name makes me think it's the latter.

Reply:
Renamed and moved to CTE